tools/lsan_blacklist.txt

```diff
@@ -0,0 +1,2 @@
+leak:__GI___strdup
+leak:__res_vinit
```

Comment:
Removed strdup and I see the __res_vinit leak with glibc 2.22-1. I added a simple opportunity to suppress all leaks from libc.so in lsan_suppressions.txt